wagtail/wagtaildocs/templates/wagtaildocs/chooser/results.html

```diff
@@ -16,5 +16,14 @@
 
     {% include "wagtailadmin/shared/pagination_nav.html" with items=documents is_ajax=1 %}
 {% else %}
-    <p>{% blocktrans %}Sorry, no documents match "<em>{{ query_string }}</em>"{% endblocktrans %}</p>
+    {% if has_documents %}
+        <p>{% blocktrans %}Sorry, no documents match "<em>{{ query_string }}</em>"{% endblocktrans %}</p>
+    {% else %}
+        {% url 'wagtaildocs:add_multiple' as wagtaildocs_add_document_url %}
+        {% if current_collection %}
```

Comment:
`current_collection` is missing from the context. It needs to be passed alongside of `has_document` from the view. e.g.
```python
return render(request, "wagtaildocs/chooser/results.html", {
    'documents': documents,
    'query_string': q,
    'is_searching': is_searching,
    'has_documents': has_documents,
    'current_collection': collection_id,
})
```